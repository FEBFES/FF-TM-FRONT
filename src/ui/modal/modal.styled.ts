import styled from 'styled-components';

export const SModalWrap = styled.div<{ show: boolean }>`
  position: fixed;
  left: 0;
  top: 0;
  right: 0;
  bottom: 0;
  background: rgba(5, 5, 5, 0.5);
  z-index: var(--z-index-modalWrap);
  display: ${({ show }) => (show ? 'block' : 'none')};
`;

export const SModalContainer = styled.div`
  z-index: var(--z-index-modalCont);
  min-width: 300px;
  min-height: 50px;
  border-radius: 14px;
  background: var(--bg-0);
  position: absolute;
  left: 50%;
  top: 50%;
  transform: translate(-50%, -50%);
  opacity: 1;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
`;
