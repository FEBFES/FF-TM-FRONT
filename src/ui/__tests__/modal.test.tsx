import React from 'react';
import {Modal} from '../modal/modal';
import renderer from 'react-test-renderer';

it('SNAPSHOTS: UI - Modal (show) component - render', () => {
    const elem = renderer.create(<Modal setShow={() => {}} show={true}>
        <div>modal</div>
    </Modal>).toJSON();
    expect(elem).toMatchSnapshot();
});