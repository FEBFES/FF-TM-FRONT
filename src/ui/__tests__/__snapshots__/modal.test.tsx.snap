// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SNAPSHOTS: UI - Modal (show) component - render 1`] = `
<div
  className="sc-bcPKhP erxbYN"
  onClick={[Function]}
  show={true}
>
  <div
    className="sc-grXZZQ iBHSSL"
    onClick={[Function]}
  >
    <div>
      modal
    </div>
  </div>
</div>
`;
